### PR TITLE
AC Automated Test - LRAC-10235,LRAC-10227 and LRAC-10234 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10235
https://issues.liferay.com/browse/LRAC-10227
https://issues.liferay.com/browse/LRAC-10234

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

**Review by manuferreira — APPROVED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsHidden.testcase [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+
+			ACSettings.goToDefinitions();
+
+			ACSettings.goToEvents();
+
+			ACNavigation.openItem(itemName = "Default Events");
+		}
+
+		task ("Check that the default event was hidden") {
+			ACDefinitions.viewHiddenEvents(customEventNameList = "${defaultEventName}");
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsHidden.testcase:107]: Hi @ViniLopes20 , this task could be:
`task ("Check that the default event is hidden") {`

**Comment by alexcandidos** [comment]
Hi @ViniLopes20 , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**